plugins/amdgpu/criu-amdgpu.proto

```diff
@@ -105,9 +105,9 @@ message criu_kfd {
 	required process_entry process_entry = 2;
 	required uint32 num_of_gpus = 3;
 	required uint32 num_of_cpus = 4;
-	repeated devinfo_entry devinfo_entries = 5;
+	repeated device_entry device_entries = 5;
```

Comment:
I guess this should have been in the previous patch.

Reply:
ACK